canvas_modules/common-canvas/src/common-properties/controls/checkboxset/checkboxset.scss

```diff
@@ -20,8 +20,21 @@
 	}
 	.bx--checkbox-wrapper { //override carbon checkbox styling TODO currently doesn't override
 		margin-bottom: $spacing-03;
+		margin-right: $spacing-03;
 	}
 	.properties-checkboxset-container {
+		.properties-checkbox-tooltip-container{
+			display: grid;
+			align-items: center;
+			grid-template-columns: 1fr 12fr;
+			width: 100%;
+		}
+		.tooltip-container{
```

Comment:
This shouldn't be needed since these this is already be set in the control-item styling.

Reply:
Removed it.